configs/virtual.js

```diff
@@ -34,12 +34,15 @@ exports.options = [
     // These are the PluginOptions passed into the plugin's
     // constructor.
     'pluginOptions': {
+      'connector': 'http://localhost:4000',
       'prefix': 'test.nerd.',
       'account': 'nerd',
       'host': 'ws://broker.hivemq.com:8000',
-      'limit': '0',
-      'max': '1000',
-      'balance': '100',
+      'minBalance': '0',
+      'maxBalance': '1000',
+      'initialBalance': '100',
+      'settleIfUnder': '0',
+      'settleIfOver': '1000',
```

Comment:
I've modified plugin virtual to have this config file, then require `ilp-plugin-tests` and test using this config. `ilp-plugin-tests` isn't in npm yet though so I'll move that stuff over the the individual plugins once it is and can be required by them.